docs/explanation/logical-axiomatization.md

```diff
@@ -0,0 +1,101 @@
+
+![](../images/discussions/logical-axiomatization/betz-pyramidal.png) 
+
+The ability of the reasoner to infer complex classes helps identify classifications that might have been missed if done manually. However, when creating an equivalent class axiom, you must be sure that it is not overly constrictive (in which case, classes that should be classified under it gets missed) nor too loose (in which case, classes will get wrongly classified under it).
+
+Example of both overly constrictive and overly loose equivalent class axiom:
+
+`neuron equivalent to cell and (part_of some 'central nervous system')`
+
+- This is overly constrictive as there are neurons outside the central nervous system (e.g. peripheral neurons).
+- This is also too loose as there are cells in the central nervous system that are not neurons (e.g. glial cells).
+
+In such cases, sometimes not having an equivalent class axioms is better (like in the case of neuron), and asserting is the best way to classify a child. 
+
+## Style guide
+
+Each ontology has certain styles and conventions in how they axiomatize. This style guide is specific to OBO ontologies. We will also give reasons as to why we choose to axiomatize in the way we do. However, be aware of your local ontology's practices. 
+
+### Respect the ontology style
+
+It is important to note that ontologies have specific axiomatization styles and may apply to, for example, selecting a preferred relation. This usually reflects their use cases. For example, the Cell Ontology has a [guide for what relations to use](https://obophenotype.github.io/cell-ontology/relations_guide/). An example of an agreement in the community is that while anatomical locations of cells are recorded using `part of`, neurons should be recorded with `has soma location`. This is to accommodate for the fact that many neurons have long reaching axons that cover multiple anatomical locations making them difficult to axiomatize using `part of`. 
+
+For example, `Betz cell`, a well known cell type which defines layer V of the primary motor cortex, synapses lower motor neurons or spinal interneurons (cell types that reside outside the brain). Having the axiom `'Betz cell' part_of 'cortical layer V'` is wrong. In this case `has soma location` is used. Because of cases like these that are common in neurons, all neurons in CL should use `has soma location`.
+
+### Avoid redundant axioms 
+
+Do not add axioms that are not required. If a parent class already has the axiom, it should not be added to the child class too. 
+For example:
+- `retinal bipolar neuron` is a child of `bipolar neuron` 
+- `bipolar neuron` has the axiom `'has characteristic' some 'cortical bipolar morphology'`
+- Therefore we do not add `'has characteristic' some 'cortical bipolar morphology'` to `retinal bipolar neuron`
+
+Axioms add lines to the ontology, resulting in larger ontologies that are harder to use. 
```

Comment:
Add:  They also add redundancy, making the ontology hard to maintain as a single change in classification might require multiple edits.)